app/src/components/integrations/elements/integrationSettings/integrationForm/utils.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { VALUE_NONE } from 'components/fields/dynamicFieldsSection/constants';
+
+export const removeNoneValues = (inputObj) => {
```

Comment:
Please move to the `'components/fields/dynamicFieldsSection/utils'` to store it near the fields and other similar utils.